pkgs/build-support/kernel/copy-modules-closure/default.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, rustPlatform, nukeReferences, kmod }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "copy-modules-closure";
+  version = "0.1.0";
+  src = ./.;
+  cargoSha256 = "sha256-EJED45EoPp2Oq7cIenM6UPicr5KxHYVOZVfVM9PwCiQ=";
```

Comment:
You could actually set `cargoVendorDir` to not having to maintain a checksum here:

```
cargoVendorDir = "vendor";
```